.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[Feature] '
+labels: 'Status:1-New, Type:Feature'
+assignees: ''
+
+---
+
+# Is your feature request related to a problem? Please describe.
+
+A clear and concise description of what the problem is.
```

Comment:
```suggestion
Describe the problem here.
```

I've found that people are more likely to be too brief rather than too long winded when creating issues. Specifying "a clear and concise" description encourages the former.